telegram/ext/filters.py

```diff
@@ -304,6 +304,15 @@ def filter(self, message: Message) -> bool:
         return False
 
 
+class _ChatType(MessageFilter):
+    def __init__(self, chat_type: str = Chat.PRIVATE):
```

Comment:
The idea was that you should also be able to pass a list of strings to e.g. filter for private *or* channel